setup.py

```diff
@@ -94,6 +94,12 @@ def install_requires():
     maintainer=find_meta('author'),
     maintainer_email=find_meta('email'),
     packages=PACKAGES,
+    entry_points={
+        'console_scripts': [
+            'gordon = gordon.main:run'
+        ],
+        'gordon.plugins': []
```

Comment:
Theoretically, core could have plugins defined within it. This is just to reserve/be explicit about the entry point namespace.